alerting/src/test/kotlin/org/opensearch/alerting/TestHelpers.kt

```diff
@@ -218,7 +216,7 @@ val NEVER_RUN = Script("return false")
 val DRYRUN_MONITOR = mapOf("dryrun" to "true")
 val TEST_HR_INDEX = "hr_data"
 val TEST_NON_HR_INDEX = "not_hr_data"
-val TEST_HR_ROLE = "hr_role"
```

Comment:
Can we keep this as `hr_role`? Is there a reason why we want to change it?

Reply:
done